web/src/main/webapp/WEB-INF/classes/org/fao/geonet/api/Messages_fre.properties

```diff
@@ -45,8 +45,8 @@ user_password_sent=Si l''utilisateur existe, vous recevrez un courriel contenant
 user_password_changed=Le mot de passe de %s a \u00E9t\u00E9 mis \u00E0 jour.
 user_password_notchanged=\u00C9chec lors du changement de mot de passe de %s. Contactez le support.
 user_password_invalid_changekey=%s est une cl\u00E9 invalide pour %s. Les cl\u00E9s ne sont valides que pendant une journ\u00E9e.
-user_with_that_email_found=Un utilisateur avec cette adresse email ou ce nom d''utilisateur existe d\u00E9j\u00E0.
-user_with_that_username_found=Un utilisateur avec cette adresse email ou ce nom d''utilisateur existe d\u00E9j\u00E0.
+user_with_that_email_found=Un utilisateur avec cette adresse email %s existe d\u00E9j\u00E0.
```

Comment:
You are reverting this change: https://github.com/geonetwork/core-geonetwork/pull/6958, it should contain the previous message.

Reply:
Reverted back to original and applied to test messages